`Unable to resolve @nx/webpack:webpack` Error on newly created `NX` repository, when invoking `npm run start`

### Current Behavior

I created a new `NX` workspace with `npx create-nx-workspace@latest` command and have selected `Standalone Node app` option with `Express [https://expressjs.com/]` preconfigured.  
When I try to run the project (after the workspace has been created and all the dependencies have been installed successfully), with no code changes, I am greeted with the error `Unable to resolve @nx/webpack:webpack`.

Here's the full log output of the command:

```sh
❯ npm run start

> test-express@0.0.0 start
> nx serve


> nx run test-express:serve:development


 >  NX   Unable to resolve @nx/webpack:webpack.

   Cannot find module '@nx/webpack/package.json'
   Require stack:
   - /home/user/repositories/test-express/node_modules/nx/src/utils/package-json.js
   - /home/user/repositories/test-express/node_modules/nx/src/utils/print-help.js
   - /home/user/repositories/test-express/node_modules/nx/src/command-line/run.js
   - /home/user/repositories/test-express/node_modules/nx/bin/run-executor.js
   Pass --verbose to see the stacktrace.


 ————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Running target serve for project test-express failed

   Failed tasks:

   - test-express:serve:development

   Hint: run the command with --verbose for more details.

   View structured, searchable error logs at https://nx.app/runs/iurX2PdBWc
```


### Expected Behavior

I should be able to start the Express server application after a clean `NX` workspace have been created with `npx create-nx-workspace@latest` command.

### GitHub Repo

https://github.com/ognjenjevremovic/test-express

### Steps to Reproduce

Steps to reproduce with the provided repository: 

1. Clone the repository,
2. Install the necessary dependencies (`npm ci`),
3. Run the project with either of commands:
  - `npm run start` or,
  - `npx nx serve test-express`.

————————————————————————————————————————————————————————————————————————————————————————————————————————————————

Steps to reproduce with the fresh `NX` workspace:

1. Create new `NX` workspace (`npx create-nx-workspace@latest`),
2. Wait for the setup to finish,
3. `cd` into the directory,
4. Run the project with either of commands:
  - `npm run start` or,
  - `npx nx serve test-express`.

### Nx Report

```shell
❯ npx nx report

 >  NX   Report complete - copy this into the issue template

   Node   : 18.12.1
   OS     : linux x64
   npm    : 8.19.2
   Hasher : Native

   nx                 : 16.0.1
   @nx/js             : 16.0.1
   @nx/jest           : 16.0.1
   @nx/linter         : 16.0.1
   @nx/workspace      : 16.0.1
   @nx/devkit         : 16.0.1
   @nx/eslint-plugin  : 16.0.1
   @nx/node           : 16.0.1
   @nrwl/tao          : 16.0.1
   nx-cloud           : 16.0.5
   typescript         : 4.9.5
```


### Failure Logs

```shell
Cannot find module '@nx/webpack/package.json'
   Require stack:
   - /home/user/repositories/test-express/node_modules/nx/src/utils/package-json.js
   - /home/user/repositories/test-express/node_modules/nx/src/utils/print-help.js
   - /home/user/repositories/test-express/node_modules/nx/src/command-line/run.js
   - /home/user/repositories/test-express/node_modules/nx/bin/run-executor.js
   Pass --verbose to see the stacktrace.


 >  NX   Running target serve for project test-express failed

   Failed tasks:

   - test-express:serve:development

   Hint: run the command with --verbose for more details.

   View structured, searchable error logs at https://nx.app/runs/iurX2PdBWc
```


### Additional Information

_No response_

--- a/docs/generated/packages/workspace/generators/preset.json
+++ b/docs/generated/packages/workspace/generators/preset.json
@@ -73,7 +73,7 @@
       "bundler": {
         "description": "The bundler to use for building the application.",
         "type": "string",
-        "enum": ["webpack", "vite", "rspack"],
+        "enum": ["webpack", "vite", "rspack", "esbuild"],
         "default": "vite"
       },
       "docker": {

--- a/packages/node/src/generators/application/application.ts
+++ b/packages/node/src/generators/application/application.ts
@@ -306,11 +306,9 @@ function addProjectDependencies(
   const frameworkDependencies = {
     express: {
       express: expressVersion,
-      '@types/express': expressTypingsVersion,
     },
     koa: {
       koa: koaVersion,
-      '@types/koa': koaTypingsVersion,
     },
     fastify: {
       fastify: fastifyVersion,

--- a/packages/workspace/src/generators/new/generate-preset.ts
+++ b/packages/workspace/src/generators/new/generate-preset.ts
@@ -150,7 +150,13 @@ function getPresetDependencies({
       };
 
     case Preset.NodeStandalone:
-      return { dependencies: {}, dev: { '@nx/node': nxVersion } };
+      return {
+        dependencies: {},
+        dev: {
+          '@nx/node': nxVersion,
+          '@nx/webpack': bundler === 'webpack' ? nxVersion : undefined,
+        },
+      };
 
     default: {
       return {

--- a/packages/workspace/src/generators/preset/preset.ts
+++ b/packages/workspace/src/generators/preset/preset.ts
@@ -150,8 +150,9 @@ async function createPreset(tree: Tree, options: Schema) {
   } else if (options.preset === Preset.NodeStandalone) {
     const { applicationGenerator: nodeApplicationGenerator } = require('@nx' +
       '/node');
+    const bundler = options.bundler === 'webpack' ? 'webpack' : 'esbuild';
     return nodeApplicationGenerator(tree, {
-      bundler: options.bundler,
+      bundler,
       name: options.name,
       linter: options.linter,
       standaloneConfig: options.standaloneConfig,

--- a/packages/workspace/src/generators/preset/schema.d.ts
+++ b/packages/workspace/src/generators/preset/schema.d.ts
@@ -10,7 +10,7 @@ export interface Schema {
   standaloneConfig?: boolean;
   framework?: string;
   packageManager?: PackageManager;
-  bundler?: 'vite' | 'webpack' | 'rspack';
+  bundler?: 'vite' | 'webpack' | 'rspack' | 'esbuild';
   docker?: boolean;
   nextAppDir?: boolean;
   routing?: boolean;

--- a/packages/workspace/src/generators/preset/schema.json
+++ b/packages/workspace/src/generators/preset/schema.json
@@ -76,7 +76,7 @@
     "bundler": {
       "description": "The bundler to use for building the application.",
       "type": "string",
-      "enum": ["webpack", "vite", "rspack"],
+      "enum": ["webpack", "vite", "rspack", "esbuild"],
       "default": "vite"
     },
     "docker": {

